Assets/Scripts/Models/Furniture.cs

```diff
@@ -280,7 +283,8 @@ private set
     }
 
     /// <summary>
-    /// Gets a list of furniture this furniture can be replaced with.
+    /// Gets a list of furniture ObjectType this furniture can be replaced with.
```

Comment:
This is actually a list of Type Tags and it is fine as a string since those are defined in the xml which modders can modify.